nimare/meta/cbma/ale.py

```diff
@@ -166,7 +179,7 @@ def _determine_histogram_bins(self, ma_maps):
         """
         if isinstance(ma_maps, list):
             ma_values = self.masker.transform(ma_maps)
-        elif isinstance(ma_maps, np.ndarray):
+        elif isinstance(ma_maps, (np.ndarray, sparse._coo.core.COO)):
```

Comment:
could you add the possible types `ma_maps` can be in the docstring of `_determine_histogram_bins`? and maybe how many dimensions the array can have.

Reply:
Done!